addons/l10n_cy/data/template/account.account-cy.csv

```diff
@@ -0,0 +1,170 @@
+"id","code","name","account_type","reconcile","name@tr","name@gr"
```

Comment:
You have translated in turkish and greec the COA but no translation for the tax report for example, any reason ?